Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -53,6 +53,11 @@ protected CacheBase()
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether context should be maintained until type has been instantiated or not.
+        /// </summary>
+        public bool MaintainContext { get; set; }
```

Comment:
@ScottIsAFool you are right.. it could be a field.. does not have to be a property